exporter/importables.go

```diff
@@ -648,8 +648,10 @@ var resourcesMap map[string]importable = map[string]importable{
 		Name: func(d *schema.ResourceData) string {
 			// TODO: if I have 2 users from different domains: test@domain1.com & test@domain2.com - then I'll generate the same name
 			// use another algorithm for name generation, like, just replace non-word characters with '_'
-			s := strings.Split(d.Get("user_name").(string), "@")
-			return s[0]
+			//s := strings.Split(d.Get("user_name").(string), "@")
+			//return s[0]
+			s := d.Get("user_name").(string)
```

Comment:
please add the command-line flag to include email domain in the naming callback, because it changes existing  behavior and thus, must be an opt-in feature.

https://github.com/databricks/terraform-provider-databricks/blob/master/exporter/command.go#L89-L95

to have that flag accessible, add import context as the first argument to a callback interface.